Makefile

```diff
@@ -1,5 +1,4 @@
 v ?= $(shell git describe --tags --abbrev=0)
-BUILDVERSION ?= $(v:v%=%)
```

Comment:
I had to look it up for myself, so thought I'd mention it here. The `BUILDVERSION` is set in the `Dockerfile`. That seems good to me. We can avoid passing it from here.